python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -157,8 +157,7 @@ def asset_wipe_command(key, **cli_args):
             confirmation = click.prompt(prompt)
 
         if confirmation == "DELETE":
-            with DagsterInstance.get() as instance:
-                instance.wipe_assets(asset_keys)
-                click.echo("Removed asset indexes from event logs")
+            instance.wipe_assets(asset_keys)
```

Comment:
Pretty sure this `DagsterInstance.get` is redundant, it is nested inside another one a few lines up.